github-repo-manager/src/main/java/com/axonivy/github/GitHubMissingCodeOfConductDetector.java

```diff
@@ -0,0 +1,34 @@
+package com.axonivy.github;
+
+import java.io.IOException;
+
+public class GitHubMissingCodeOfConductDetector extends GitHubFileDetector {
+
+	public static void main(String[] args) throws IOException {
```

Comment:
In which build is this class called?
You should call it in a build like here:
https://github.com/axonivy/github-repo-manager/tree/master/build/public-repo-license